### PR TITLE
GG-24887 Test valid loading of JDBC driver list.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/e2e/testcafe/mocks/agentTasks.js [hunk 1]
```diff
@@ -211,3 +211,17 @@ export const agentStat = (clusters) => (ws) => {
 export const errorResponseForEventType = (eventType) => (ws) => {
     ws.errorOn(eventType, () => DFLT_FAILURE_RESPONSE);
 };
+
+export const TEST_JDBC_IMPORT_DATA = {
+    drivers: [{
+        'jdbcDriverJar': 'test-driver.jar',
+        'jdbcDriverClass': 'test.dvired.cls',
```
Comment (MaxLosevskoy) [inline modules/web-console/e2e/testcafe/mocks/agentTasks.js]: Is that **dvired** correct?
Or it is not important?

Comment (vsisko) [reply modules/web-console/e2e/testcafe/mocks/agentTasks.js]: It is not important. Test of the import require a real configured database or should be fully mocked.

File: modules/web-console/frontend/app/modules/agent/AgentManager.service.ts [hunk 2]
```diff
@@ -459,6 +459,8 @@ export default class AgentManager {
                 if (__dbg)
                     console.log('Received response: ', evt);
 
+                console.log(evt);
```
Comment (Klaster1) [inline modules/web-console/frontend/app/modules/agent/AgentManager.service.ts]: Please remove.

Comment (vsisko) [reply modules/web-console/frontend/app/modules/agent/AgentManager.service.ts]: Removed.

**Pull request merged**